docs/sources/helm-charts/mimir-distributed/get-started-helm-charts/_index.md

```diff
@@ -138,12 +138,25 @@ Using a custom namespace solves problems later on because you do not have to ove
 
 ## Generate some metrics for testing
 
-The Grafana Mimir Helm chart can collect metrics or logs, or both, about Grafana Mimir itself. This is called _metamonitoring_.
+The Grafana Mimir Helm chart can collect metrics, logs, or both, about Grafana Mimir itself. This is called _metamonitoring_.
 In the example that follows, metamonitoring scrapes metrics about Grafana Mimir itself, and then writes those metrics to the same Grafana Mimir instance.
 
-1. Create a YAML file called `custom.yaml` for your Helm values.
+1. Download the Grafana Agent Operator Custom Resource Definitions (CRDs) from
+   https://github.com/grafana/agent/tree/main/production/operator/crds.
```

Comment:
Is it ok to have the dot at the end of the URL? It might be a GitHub quirk, but when I click this link in my browser I get taken to a URL which has the dot at the end and get a 404. Have you tried rendering the site locally with `make docs`?

Reply:
> Is it ok to have the dot at the end of the URL? It might be a GitHub quirk, but when I click this link in my browser I get taken to a URL which has the dot at the end and get a 404. Have you tried rendering the site locally with `make docs`?

We optimize for https://grafana.com; are you seeing this 404 from GitHub directly?

Reply:
I cannot reproduce this issue when I run `make docs` locally; I get redirected to https://github.com/grafana/agent/tree/main/production/operator/crds.

Reply:
this redirects just fine in the GitHub UI for me; I viewed it in the rendered Markdown view, not a raw or diffed view.

`make docs` doesn't work for me locally for reasons I haven't figured out yet, but I run `grip` which uses the github markdown rendering API to check for this